src/main/java/org/testng/TestRunner.java

```diff
@@ -542,6 +543,14 @@ private void initMethods() {
             comparator);
   }
 
+  private ITestNGMethod[] buildBeforeClassMethods(Map<String, ITestNGMethod[]> beforeClassConfig) {
```

Comment:
Could be moved into `ITestClassConfigInfo` by replacing the params with a call to `getBeforeClassConfig`

Reply:
Sure, this method was useless